src/addon_transport/http_transport/legacy/mod.rs

```diff
@@ -213,6 +215,14 @@ fn build_legacy_req(transport_url: &Url, path: &ResourcePath) -> Result<Request<
             let video_size = path
                 .get_extra_first_value(VIDEO_SIZE_EXTRA_PROP.name.as_str())
                 .and_then(|video_size| video_size.parse().ok());
+            let video_filename =
```

Comment:
Probably this should be below the `if let Some(video_size) = video_size {` block, to not split the logic blocks up.